src/ssml-types/audio-library.ts

```diff
@@ -36,7 +36,7 @@ const getSsml = (data: LibraryData) => {
 export default {
   getTimelineHtml: (data: LibraryData) => {
     return `<strong class="audio-description">
-            <img src="./images/library.png" />
+            <i class="material-icons">library_music</i>
```

Comment:
Would you mind reverting this change and moving to a separate PR? That would make this one a bit more contained.

Reply:
@Fleker ok reverted that commit and will PR from a branch https://github.com/GWS-Technologies-LTD/nightingale-ssml-editor/tree/google-icons